### DISCUSSION TITLE
How do I manually update a package?

### DISCUSSION THREAD

**Discussion by DougRogers** [microsoft/vcpkg#22281] [Q&A]
There are some packages that are out of date and are waiting to be updated formally by Microsoft.  Is there a way to manually update a package to use the latest release from github?

**Comment by autoantwort** [comment]
You can fork this repo and update everything you want

**Comment by mathisloge** [comment]
instead of a complete fork you can also use [custom-registries](https://github.com/microsoft/vcpkg/blob/master/docs/users/registries.md) or [overlay ports](https://github.com/microsoft/vcpkg/blob/master/docs/specifications/ports-overlay.md)
